catalog/core/catalog-core-api-impl/src/test/java/ddf/catalog/data/impl/EmptyMetacardType.java

```diff
@@ -42,4 +41,21 @@ public AttributeDescriptor getAttributeDescriptor(String attributeName) {
         return null;
     }
 
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+
+        return true;
+    }
+
+    @Override
+    public int hashCode() {
+        return 31;
```

Comment:
In case some of you wonder why 31, here's a nice explanation: http://stackoverflow.com/questions/299304/why-does-javas-hashcode-in-string-use-31-as-a-multiplier.